utils/users.js

```diff
@@ -1,11 +1,22 @@
 const { fetchUser } = require("../models/users");
+const firestore = require("../utils/firestore");
+const userModel = firestore.collection("users");
+
+const addUserToDBForTest = async (userData) => {
+  try {
+    await userModel.add(userData);
+  } catch (err) {
+    // console.log(err);
```

Comment:
why aren't you handling error here?

Reply:
as discussed in the call, removing try catch